src/lib/core/port.h

```diff
@@ -133,6 +134,16 @@ struct port_vtab {
 	 * implementation
 	 */
 	struct Mem *(*get_vdbemem)(struct port *port, uint32_t *size);
+	/**
+	 * Get the content of a port as a list of port_c_entries.
```

Comment:
`port_c_entry`, not `port_c_entries`. Also, if it returns a list, it should be called `get_c_entries`.

Reply:
Done.